src/main/resources/assets/fabriblocks/lang/zh_cn.json

```diff
@@ -0,0 +1,67 @@
+{
+	"block.fabriblocks.scute_block": "鳞甲块",
+	"block.fabriblocks.bamboo_block": "竹块",
+	"block.fabriblocks.granite_tiling": "花岗岩瓦砖",
+	"block.fabriblocks.diorite_tiling": "闪长岩瓦砖",
+	"block.fabriblocks.andesite_tiling": "安山岩瓦砖",
+	"block.fabriblocks.granite_diorite_tiling": "花岗岩和闪长岩瓦砖",
+	"block.fabriblocks.diorite_andesite_tiling": "闪长岩和安山岩瓦砖",
+	"block.fabriblocks.andesite_granite_tiling": "安山岩和花岗岩瓦砖",
+	"block.fabriblocks.decorated_sandstone": "装饰性砂岩",
+	"block.fabriblocks.nether_brick_pillar": "地狱砖柱",
+	"block.fabriblocks.red_nether_brick_pillar": "红色地狱砖柱",
+	"block.minecraft.bookshelf": "橡木书架",
+	"block.fabriblocks.spruce_bookshelf": "云杉木书架",
+	"block.fabriblocks.birch_bookshelf": "白桦木书架",
+	"block.fabriblocks.jungle_bookshelf": "丛林木书架",
+	"block.fabriblocks.acacia_bookshelf": "金合欢木书架",
+	"block.fabriblocks.dark_oak_bookshelf": "深色橡木书架",
+	"block.fabriblocks.mossy_bricks": "苔砖块",
+	"block.fabriblocks.mossy_brick_stairs": "苔砖楼梯",
+	"block.fabriblocks.mossy_brick_slab": "苔砖台阶",
+	"block.fabriblocks.snow_bricks": "雪砖",
+	"block.fabriblocks.snow_brick_stairs": "雪砖楼梯",
+	"block.fabriblocks.snow_brick_slab": "雪砖台阶",
+	"block.fabriblocks.smooth_stone_stairs": "平滑石楼梯",
+	"block.fabriblocks.cut_sandstone_stairs": "切制砂岩楼梯",
+	"block.fabriblocks.cut_red_sandstone_stairs": "切制红砂岩楼梯",
+	"block.fabriblocks.cracked_stone_brick_stairs": "裂石砖楼梯",
+	"block.fabriblocks.cracked_stone_brick_slab": "裂石砖台阶",
+	"block.fabriblocks.oak_leaf_carpet": "橡木叶地毯",
+	"block.fabriblocks.spruce_leaf_carpet": "云杉木叶地毯",
+	"block.fabriblocks.birch_leaf_carpet": "白桦木叶地毯",
+	"block.fabriblocks.jungle_leaf_carpet": "丛林木叶地毯",
+	"block.fabriblocks.acacia_leaf_carpet": "金合欢木叶地毯",
+	"block.fabriblocks.dark_oak_leaf_carpet": "深色橡木叶地毯",
+	"block.fabriblocks.red_nether_brick_fence": "红地狱砖栅栏",
+	"block.fabriblocks.stone_wall": "石墙",
+	"block.fabriblocks.smooth_stone_wall": "平滑石墙",
+	"block.fabriblocks.smooth_sandstone_wall": "平滑砂岩墙",
+	"block.fabriblocks.smooth_red_sandstone_wall": "平滑红砂岩墙",
+	"block.fabriblocks.smooth_quartz_wall": "平滑石英墙",
+	"block.fabriblocks.polished_granite_wall": "磨制花岗岩墙",
+	"block.fabriblocks.polished_diorite_wall": "磨制闪长岩墙",
+	"block.fabriblocks.polished_andesite_wall": "磨制安山岩墙",
+	"block.fabriblocks.cracked_stone_brick_wall": "裂石砖墙",
+	"block.fabriblocks.quartz_wall": "石英墙",
+	"block.fabriblocks.prismarine_brick_wall": "海晶石砖墙",
+	"block.fabriblocks.dark_prismarine_wall": "深色海晶石墙",
+	"block.fabriblocks.purpur_wall": "紫珀墙",
+	"block.fabriblocks.mossy_brick_wall": "苔砖墙",
+	"block.fabriblocks.snow_brick_wall": "雪砖墙",
+	"block.minecraft.lectern": "橡木讲台",
+	"block.fabriblocks.spruce_lectern": "云杉木讲台",
+	"block.fabriblocks.birch_lectern": "白桦木讲台",
+	"block.fabriblocks.jungle_lectern": "丛林木讲台",
+	"block.fabriblocks.acacia_lectern": "金合欢木讲台",
+	"block.fabriblocks.dark_oak_lectern": "深色橡木讲台",
+	"block.fabriblocks.granite_pressure_plate": "花岗岩压力板",
+	"block.fabriblocks.diorite_pressure_plate": "花岗岩压力板",
```

Comment:
It's the same translation as for granite pressure plate ^^

Reply:
I have fixed these two bugs :)